Makefile

```diff
@@ -214,7 +217,7 @@ helm-docs: $(HELM_DOCS) ## Use helm-docs to generate documentation from helm cha
 docs-preview: ## Preview the documentation through mkdocs
 	mkdocs serve
 
-docs-build:  ## Build the documentation to the `site/` directory
+docs-build: docs ## Build the documentation to the `site/` directory
```

Comment:
@galexrt : does this seem good to you, or would you prefer this be absent? I told Michael to include it for us to chat about here.

Reply:
Seems like we don't need it here. The docs-check.yml CI action calls both `make docs` (with the change above), and `make docs-build`, so we shouldn't need to generate the example docs from both actions.

Reply:
@travisn  wrote:
> Seems like we don't need it here. The docs-check.yml CI action calls both `make docs` (with the change above), and `make docs-build`, so we shouldn't need to generate the example docs from both actions.

what is your suggestion? remove the pre-existing invocation of `make docs-build` from the workflow?

Reply:
I'd suggest we revert this line. `make docs-build` is for the purpose of checking published documentation to rook.io, rather than just generating local docs that is already covered with `make docs`.

Reply:
> I'd suggest we revert this line. `make docs-build` is for the purpose of checking published documentation to rook.io, rather than just generating local docs that is already covered with `make docs`.

done: I removed the dependency